courses/views.py

```
@@ -0,0 +1,38 @@
+
+def course_detail(request, slug):
+    course = get_object_or_404(Course, slug=slug, is_active=True)
+    return render(request, 'courses/course_detail.html', {'course': course})
+
+@login_required
+def enroll_course(request, slug):
+    course = get_object_or_404(Course, slug=slug, is_active=True)
+    enrollment, created = Enrollment.objects.get_or_create(
+        student=request.user,
+        course=course,
+        defaults={'is_active': True}
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)